apps/pom.xml

```diff
@@ -615,6 +615,7 @@ checkJavaFormat
 						<!-- Only look in source directories, basically ignores generated code -->
 						<sourceDirectories>
 							<sourceDirectory>${project.build.sourceDirectory}</sourceDirectory>
+							<sourceDirectory>${project.build.testSourceDirectory}</sourceDirectory>
 						</sourceDirectories>
```

Comment:
This is the main thing that was fixed; in order to properly exclude generated files from checkstyle, I set up an inclusion to only allow source files last time I messed with this. However, I neglected to add the test source files here as well, essentially disabling checkstyle for test classes.

The rest were just errors that arose since last time tests classes have been checked.